Tasks/MavenAuthenticateV0/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "maven-authenticate-task",
```

Comment:
nit: for consistency across auth tasks, you can name it mavenauthenticate.

Reply:
Fixed